modules/gdscript/language_server/gdscript_language_server.cpp

```diff
@@ -36,6 +36,8 @@
 #include "editor/editor_node.h"
 #include "editor/editor_settings.h"
 
+int GDScriptLanguageServer::port_override = -1;
+
 GDScriptLanguageServer::GDScriptLanguageServer() {
 	_EDITOR_DEF("network/language_server/remote_host", host);
 	_EDITOR_DEF("network/language_server/remote_port", port);
```

Comment:
Is it not possible to set `"network/language_server/remote_port"` to `port_override > -1 ? port_override : port` here so that later you don't need to do `if/else` twice to check for override. 

(Ideally `_EDITOR_DEF` could be set in `main.cpp` during the option handling, but seems like it is not supported there, as I understand.)

Reply:
Yep! I forget certain C++ features exist sometimes lol

And yeah, _EDITOR_DEF was what I was looking for at first, but it doesn't work in main.cpp afaik.

Also, does _EDITOR_DEF define an editor setting only for the instance running? We're trying to be able to spin up multiple language server instances with different ports at the same time, without affecting the user's editor settings.

Reply:
Yeah, makes sense, I do see that editor settings are saved and that would affect multiple instances. So can "resolve" this comment.